CONTRIBUTING.md

```diff
@@ -79,21 +53,16 @@ to:
 * [agreement@artefactual.com](mailto:agreement@artefactual.com) 
 
 **If you are working on a larger pull request and/or new feature**, please be
-sure to read this page:
-
-* https://wiki.accesstomemory.org/Development/Recommendations
+sure to read [our recommendations](https://wiki.accesstomemory.org/Development/Recommendations)
```

Comment:
Again, I would suggest that this page be reviewed so it matches current recommendations